docs/stories/components/Accordion.mdx

```diff
@@ -0,0 +1,119 @@
+import { Meta, Canvas, ArgTypes } from '@storybook/blocks';
+import { Accordion } from '@strapi/design-system';
+
+import * as AccordionStories from './Accordion.stories';
+
+<Meta of={AccordionStories} />
+
+# Accordion
+
+- [Overview](#overview)
+- [Usage](#usage)
+- [Props](#props)
+- [Variants](#variants)
+- [Accessibility](#accessibility)
+
+## Overview
+
+A vertically stacked set of interactive headings that each reveal an associated section of content. Accordion is a
+composite component designed to be flexible and versitile to display whatever content you require.
+
+<ViewSource path="components/Accordion" />
+
+<Canvas of={AccordionStories.Base} />
+
+## Imports
+
+```js
+import { Accordion } from '@strapi/design-system';
+```
+
+## Props
+
+### Accordion
+
+<ArgTypes of={Accordion.Root} />
```

Comment:
I'm seeing this in the story here http://localhost:6006/?path=/docs/components-accordion--docs

Reply:
yeah they dont work yet :( I need to make a custom component & learn how to do it... SOON